frontend/src/components/AddMember/index.js

```diff
@@ -1,98 +1,163 @@
+      ],
+      male: [
+        { text: "", value: "" },
+        { text: "Bunic", value: "1" },
+        { text: "Tată", value: "2" },
+        { text: "Fiu", value: "3" },
+        { text: "Frate", value: "4" },
+        { text: "Alt membru al familiei", value: "5" },
+        { text: "Persoană în grijă", value: "6" }
+      ]
+    },
+    yesNo: [
+      { key: "true", value: "Da" },
+      { key: "false", value: "Nu" }
+    ],
+    preexistingConditions: [
+      { value: "1", text: "Boală cardiovasculară" },
+      { value: "2", text: "Diabet" },
+      { value: "3", text: "Boală pulmonară" },
+      { value: "4", text: "Cancer" },
+      { value: "5", text: "Altă boală cronică" }
+    ],
+    isolation: [
+      { key: "1", value: "Auto izolare" },
+      { key: "2", value: "Carantină la domiciliu" },
+      { key: "3", value: "Carantină specializată" },
+      { key: "4", value: "Niciuna" }
+    ]
+  };
 
-  const options2 = [
-    { key: "1", value: "Auto izolare" },
-    { key: "2", value: "Carantină la domiciliu" },
-    { key: "3", value: "Carantină specializată" },
-    { key: "4", value: "Niciuna" }
-  ];
+  const toggleSex = event => {
+    setFemale(event.target.value === "Feminin");
+  };
 
-  const genderOptions = [
-    { key: "", text: "" },
-    { key: "1", text: "Feminin" },
-    { key: "2", text: "Masculin" }
-  ];
+  const togglePreexistingConditions = value => {
+    setPreexistingConditions(value === "Da");
+  };
 
-  var ageOptions = [...Array(100).keys()].map(index => ({
-    text: index,
-    value: index
-  }));
-  ageOptions[0].text = "";
+  const toggleDisabilities = value => {
+    setDisabilities(value === "Da");
+  };
+
+  const validateAge = event => {
+    const value = parseInt(event.target.value.replace(/[^0-9]+/g, ""));
+    // age range is between 0 and 110 - might be worth reconsidering?
+    const boundedValue = Math.min(110, Math.max(0, value));
+
+    event.target.value = isNaN(boundedValue) ? "" : boundedValue;
+  };
 
   return (
     <SidebarLayout>
       <form>
-        <Hero
-          title="Creează cont pentru un membru al familiei"
-          subtitle="Înregistrează un membru al familiei care nu poate accesa platforma și ajută-l să își facă evaluarea zilnică a simptomelor COVID-19."
-        />
-        <Input label="Nume și prenume" type="text" required="true" />
-        <div className="columns">
-          <div className="column is-6">
-            <Select label="Legătura familiala" options={relationshipOptions} />
+        <div className="add-member">
+          <Hero
+            title="Creează cont pentru un membru al familiei"
+            subtitle="Înregistrează un membru al familiei care nu poate accesa platforma și ajută-l să își facă evaluarea zilnică a simptomelor COVID-19."
+          />
+          <Input label="Nume și prenume" type="text" required={true} />
+          <div className="columns">
+            <div className="column is-3">
+              <Input
```

Comment:
I've added a ticket on taskforce component for an input number (https://github.com/code4romania/taskforce-fe-components/issues/135). This would have been awesome here ...